component/notification/src/main/java/org/exoplatform/social/notification/impl/SpaceWebNotificationServiceImpl.java

```diff
@@ -208,6 +222,20 @@ public Map<String, Long> countUnreadItemsByApplication(long userIdentityId, long
     return metadataService.countMetadataItemsByMetadataTypeAndAudienceId(METADATA_TYPE_NAME, userIdentityId, spaceId);
   }
 
+  @Override
+  public Map<Long, Long> countUnreadItemsBySpace(String authenticatedUser) {
```

Comment:
In service layer, we don't need to know whether the user is authenticated or not, the API has to be more clear about it. By naming the parameter `authenticatedUser`, it like a precondition to be able to execute this method to be authenticated with that user, while it's just a parameter to use about the username to apply a search query on it. (By using `username` instead of `userId`, you will be more specific as well to know that it's the user name and not the user identity id)

```suggestion
  public Map<Long, Long> countUnreadItemsBySpace(String username) {
```